src/System.Drawing.Common/tests/Drawing2D/GraphicsPathIteratorTests.cs

```diff
@@ -0,0 +1,426 @@
+        public void Enumerate_NullPointsTypes_ThrowsNullReferenceException(PointF[] points, byte[] types)
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (GraphicsPathIterator gpi = new GraphicsPathIterator(gp))
+            {
+                Assert.Throws<NullReferenceException>(() => gpi.Enumerate(ref points, ref types));
+            }
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [MemberData(nameof(PointsTypesLenghtMismatch_TestData))]
+        public void CopyData_PointsTypesMismatch_ThrowsArgumentException(PointF[] points, byte[] types)
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (GraphicsPathIterator gpi = new GraphicsPathIterator(gp))
+            {
+                AssertExtensions.Throws<ArgumentException>(null, () => gpi.CopyData(ref points, ref types, 0, points.Length));
+            }
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [MemberData(nameof(NullPointsTypes_TestData))]
+        public void CopyData_NullPointsTypes_ThrowsNullReferenceException(PointF[] points, byte[] types)
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (GraphicsPathIterator gpi = new GraphicsPathIterator(gp))
+            {
+                Assert.Throws<NullReferenceException>(() => gpi.CopyData(ref points, ref types, 0, 1));
+            }
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [InlineData(-1, 2)]
+        [InlineData(0, 3)]
+        public void CopyData_StartEndIndexesOutOfRange_ThrowsArgumentException(int startIndex, int endIndex)
+        {
+            PointF[] resultPoints = new PointF[0];
+            byte[] resultTypes = new byte[0];
+
+            using (GraphicsPath gp = new GraphicsPath())
+            using (GraphicsPathIterator gpi = new GraphicsPathIterator(gp))
+            {
+                AssertExtensions.Throws<ArgumentException>(null, () => gpi.CopyData(ref resultPoints, ref resultTypes, startIndex, endIndex));
+            }
+        }
+
+        //public static IEnumerable<object[]> CopyData_StartEndIndexesOutOfRange_TestData()
+        //{
+        //    yield return new object[] { new PointF[3], new byte[3], int.MinValue, 2 };
+        //    yield return new object[] { new PointF[3], new byte[3], 0, int.MaxValue };
+        //    yield return new object[] { new PointF[3], new byte[3], 2, 0 };
+        //}
+
+        //[ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        //[MemberData(nameof(CopyData_StartEndIndexesOutOfRange_TestData))]
+        //public void CopyData_StartEndIndexesOutOfRange_ReturnsExpeced(PointF[] points, byte[] types, int startIndex, int endIndex)
```

Comment:
Can you uncomment this, and instead add an `[ActiveIssue(22026)]` attribute? This will prevent it from running, but will still compile the code.